src/ASM/ASMs2D.C

```diff
@@ -181,12 +181,15 @@ bool ASMs2D::read (std::istream& is)
 }
 
 
-bool ASMs2D::write (std::ostream& os, int) const
+bool ASMs2D::write (std::ostream& os, int basis) const
```

Comment:
maybe writeBasis would be a better suited name now...

Reply:
we can consider renaming that but i'd rather do that separately. this is a virtual all the way down in ASMbase..